Sources/SmileID/Classes/SmileID.swift

```diff
@@ -57,14 +60,24 @@ public class SmileID {
         return SmileView(initialDestination: destination).environmentObject(navigationState)
     }
 
-    public class func documentVerificationScreen(userId _: String = "user-\(UUID().uuidString)",
-                                                 jobId _: String = "job-\(UUID().uuidString)",
-                                                 showAttribution _: Bool = true,
-                                                 showInstruction _: Bool = true,
+    public class func documentVerificationScreen(userId: String = "user-\(UUID().uuidString)",
```

Comment:
The partner should also be able to pass in a selfie file, if they want to use that directly instead of performing selfie capture